python-nitrate.spec

```diff
@@ -1,6 +1,6 @@
 Name: python-nitrate
 Version: 1.6
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
The `Release` field is used for spec file changes only. Please bump the `Version` field to `1.7` instead.

Reply:
Bumped to 1.7-1